### PR TITLE
30794: Split current compensation question into two questions

### PR DESCRIPTION
**Pull request by craigwheeler**
## Description
Updates to split current compensation question into two questions and update design and copy

## Original issue(s)
department-of-veterans-affairs/va.gov-team#30794

## Screenshots
Apply for Health Care  Veterans Affairs 2021-11-08 13-29-51

## Acceptance criteria
- [x] compensation question should be splint into two questions
- [x]  updated design and copy

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/applications/hca/components/CustomReviewField.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const CustomReviewField = ({
+  children: {
+    props: { uiSchema, formData },
+  },
+}) => (
+  <div className="review-row">
```
Comment (timwright12) [inline src/applications/hca/components/CustomReviewField.jsx:8]: Needs to be a `dl>`

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: src/applications/hca/components/CustomReviewField.jsx (hunk 1 above)
Comment (Mottie) [reply src/applications/hca/components/CustomReviewField.jsx:8]: I think the wrapper of this component is a `<dl>`. Axe doesn't throw any errors if there is a single `<div>` wrapping the `<dt>` & `<dd>` children.

Comment (craigwheeler) [reply src/applications/hca/components/CustomReviewField.jsx:8]: hey @timwright12 👋  ... thanks for the call out! So I looked into using the `<dl>` tag here but looks like the review page already provides one so were using a `div` to wrap the name value group for the `review-row` styling here ... https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#wrapping_name-value_groups_in_div_elements verified the elements pass accessibility checks as well


Apply for Health Care  Veterans Affairs 2021-11-09 09-21-56

**Comment by Dene-human** [comment]
Looks great! One feedback:
- The VA pension question is **required** (if shown)

**Review by timwright12 — APPROVED** [review]

**Review by LinnJS — APPROVED** [review]
🚀 great work @craigwheeler

File: src/applications/hca/components/CustomReviewField.jsx (hunk 1 above)
Comment (LinnJS) [reply src/applications/hca/components/CustomReviewField.jsx:8]: @timwright12 @Mottie thanks for the informative review, always learn something new

**Comment by craigwheeler** [comment]
> Looks great! One feedback:
> 
> * The VA pension question is **required** (if shown)

hey @Dene-human updated the pension field to be `required` lemme know if there's anything else I can update thanks!

**Comment by Dene-human** [comment]
RI checked. Looks good by me. Thanks!

**Pull request merged**